code/modules/surgery/organs/internal/stomach/_stomach.dm

```diff
@@ -312,15 +322,12 @@
 	emp_vulnerability = 20
 	metabolism_efficiency = 0.1
 
-/obj/item/organ/internal/stomach/cybernetic/emp_act(severity)
-	. = ..()
-	if(. & EMP_PROTECT_SELF)
-		return
-	if(!COOLDOWN_FINISHED(src, severe_cooldown)) //So we cant just spam emp to kill people.
-		owner.vomit(stun = FALSE)
-		COOLDOWN_START(src, severe_cooldown, 10 SECONDS)
-	if(prob(emp_vulnerability/severity)) //Chance of permanent effects
-		organ_flags |= ORGAN_EMP //Starts organ faliure - gonna need replacing soon.
-
+/obj/item/organ/internal/stomach/cybernetic/surplus
+	name = "surplus cybernetic stomach"
+	desc = "A terribly ineffecient mockery of a human stomach. Offers no protection against EMPs."
```

Comment:
```suggestion
	desc = "A mechanical plastic oval that utilizes sulfuric acid instead of stomach acid. Offers no protection against EMPs."
```

It'd be fun if stomachs interacted with the concept of stomach acid, imagine barfing up with a prosthetic stomach dealing burn damage because its stomach acid substitute isn't properly diluted by saliva (yes, that's why you don't die from vomiting IRL)

Reply:
feature creep!@!

(not gonna happen for now)